src/components/User/SystemAssets/Issuance/components/IssuanceHistory.vue

```diff
@@ -99,17 +89,31 @@
           </div>
         </div>
       </template>
+
+      <div class="app__more-btn-wrp">
+        <collection-loader
+          :first-page-loader="getList"
+          @first-page-load="setList"
+          @next-page-load="onMoreButtonClick"
+          ref="collectionLoaderBtn"
+        />
+      </div>
     </div>
   </div>
 </template>
 
 <script>
 import IssuanceMixin from './issuance.mixin'
 import { EmailGetter } from '@comcom/getters'
+import Bus from '@/utils/EventBus'
 
 export default {
   mixins: [IssuanceMixin],
-  components: { EmailGetter }
+  components: { EmailGetter },
+
+  async created () {
+    Bus.$on('issuance:updateRequestList', _ => { this.$refs.collectionLoaderBtn.loadFirstPage() })
```

Comment:
the issue about 80 chars

Reply:
are you sure we show request list here? the component named `IssuanceHistory`, please recheck we do not show history instead of requests here.